ROADMAP.md

```diff
@@ -8,43 +8,24 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 See the [Github project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
+## Rook Ceph 1.8
 
-## Rook 1.7
-
-The following high level features are targeted for Rook v1.7 (July 2021). For more detailed project tracking see the [v1.7 board](https://github.com/rook/rook/projects/21).
-
-* Ceph
-  * Helm chart for the cluster CR [#2109](https://github.com/rook/rook/issues/2109)
-  * Configure bucket notifications with a CRD ([design doc](https://github.com/rook/rook/blob/master/design/ceph/object/ceph-bucket-notification-crd.md))
-  * Add alpha support for COSI (Container object storage interface) with K8s 1.22 [#7843](https://github.com/rook/rook/issues/7843)
-  * Disaster Recovery (DR): CSI solution for application failover in the event of cluster failure
-  * Allow OSDs on PVCs to automatically grow when the cluster is nearly full [#6101](https://github.com/rook/rook/issues/6101)
-  * OSD encryption key rotation [#7925](https://github.com/rook/rook/issues/7925)
-  * iSCSI gateway deployment [#4334](https://github.com/rook/rook/issues/4334)
-  * Use go-ceph to interact with object store instead of `radosgw-admin` [#7924](https://github.com/rook/rook/issues/7924)
-  * RGW Multi-site replication improvements towards declaring the feature stable [#6401](https://github.com/rook/rook/issues/6401)
-  * More complete solution for protecting against accidental cluster deletion [#7885](https://github.com/rook/rook/pull/7885)
-  * Remove support for Nautilus, focusing on support for Octopus and Pacific [#7908](https://github.com/rook/rook/issues/7908)
- * Build hygiene
-  * Complete conversion from Jenkins pipeline to GitHub actions
+The following high level features are targeted for Rook v1.8 (December 2021). For more detailed project tracking see the [v1.8 board](https://github.com/rook/rook/projects/23).
+
+* Configure bucket notifications with a CRD ([design doc](https://github.com/rook/rook/blob/master/design/ceph/object/ceph-bucket-notification-crd.md))
+* Add alpha support for COSI (Container object storage interface) with K8s 1.22 [#7843](https://github.com/rook/rook/issues/7843)
```

Comment:
@thotz is this realistic in your opinion?

Reply:
I'll attent the COSI meeting today and see when COSI expects to release also. I don't see anything that appears COSI related in the release notes, so I'm not sure it actually made it into K8s 1.22.

Reply:
it got moved to k8s 1.23.

Reply:
Ok I'll move it to the section below since it's not clearly in 1.8